### PR TITLE
Remove es_port and prom_port variables

### PR DESCRIPTION
**Pull request by rsevilla87**
Depends-On: 438

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: snafu/hammerdb/hammerdb_wrapper.py [hunk 1]
```diff
@@ -58,10 +56,6 @@ def __init__(self, parser):
         self.args.db_postgresql_storedprocs = ""
 
         # exporting the generic settings
-        if "es_server" in os.environ:
```
Comment (dry923) [inline snafu/hammerdb/hammerdb_wrapper.py:61]: do we need to define self.args.es_server from this still?

Comment (rsevilla87) [reply snafu/hammerdb/hammerdb_wrapper.py:61]: As we discussed in slack, this is not required and I think it belongs to an old indexing implementation

**Review by dry923 — APPROVED** [review]
LGTM!

**Comment by dry923** [comment]
@rsevilla87 needs a rebase

**Review by acalhounRH — CHANGES_REQUESTED** [review]
comments inline.

**Comment by jtaleric** [comment]
@rsevilla87 Checking in on addressing @acalhounRH's comments.

File: snafu/run_snafu.py [hunk 2]
```diff
@@ -60,44 +61,33 @@ def main():
...
-        if os.environ["es_port"] != "":
-            es['port'] = os.environ["es_port"]
-            logger.info("Using elasticsearch server with port:" + es['port'])
-    es_verify_cert = os.getenv("es_verify_cert", "true")
-    if len(es.keys()) == 2:
-        if os.environ["es_index"] != "":
-            index_args.prefix = os.environ["es_index"]
-            logger.info("Using index prefix for ES:" + index_args.prefix)
+    es_settings = {}
+    es_settings["server"] = os.getenv("es")
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply snafu/run_snafu.py:65]: This check is made by the line:
```
     if es_settings["server"]: 
```
which checks if the server env var was set.

File: snafu/run_snafu.py [hunk 3]
```diff
@@ -60,44 +61,33 @@ def main():
...
             logger.info(json.dumps(es.info(), indent=4))
-
         except Exception as e:
-            logger.warn("Elasticsearch connection caused an exception : %s" % e)
+            logger.warn("Elasticsearch connection caused an exception: %s" % e)
             index_args.index_results = False
 
     index_args.document_size_capacity_bytes = 0
     if index_args.index_results:
-        # call py es bulk using a process generator to feed it ES documents
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply snafu/run_snafu.py:99]: Rolled back

File: snafu/run_snafu.py [hunk 4]
```diff
@@ -183,54 +173,41 @@ def get_valid_es_document(action, index, index_args):
...
-
-            es_valid_document = get_valid_es_document(prometheus_doc,
-                                                      "prometheus_data",
-                                                      index_args)
-
+            es_valid_document = get_valid_es_document(prometheus_doc, "prometheus_data", index_args)
             yield es_valid_document
 
-    es = {}
-    if "prom_es" in os.environ:
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply snafu/run_snafu.py:199]: Same as above. This check is made by the line
```
    if es_settings["server"]::
```

**Review by jtaleric — APPROVED** [review]
LGTM

**Comment by rsevilla87** [comment]
LGTM

**Pull request merged**